sys-kernel/coreos-sources/files/4.4/0015-Overlayfs-Use-copy-up-security-hooks.patch

```diff
@@ -1,4 +1,4 @@
-From 64ef0efdd90f5aae4fae7c76783b09af53d29dfe Mon Sep 17 00:00:00 2001
+From 78e4150dc214a67ee53b9d2fc0a3d893286d21eb Mon Sep 17 00:00:00 2001
```

Comment:
Rebased on to the patch formally  known as 0019-overlayfs-use-a-minimal-buffer-in-ovl_copy_xattr.patch I presume.